app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        command, original_file, copy_of_file = command.split()
+    except ValueError:
+        return
+    if command != "cp":
```

Comment:
Unite conditions into one? There're the same return blocks

Reply:
I don't think that it is a good practice because on big code it will be hard to read. But I will apply your changes.